chsdi/lib/sphinxapi/sphinxapi.py

```diff
@@ -19,7 +19,28 @@
 import socket
 import re
 from struct import pack, unpack
+import six
+
+if six.PY3:
+    long = int
```

Comment:
I must say it at least once : overriding reserved words / types is something you should never do. ^_^